LION/models/learned_regularizer/AR.py

```diff
@@ -0,0 +1,83 @@
+# This file is part of LION library
+# License : BSD-3
+#
+# Author  : Zakhar Shumaylov, Subhadip Mukherjee
+# Modifications: Ander Biguri, Zakhar Shumaylov, Charlie Shoebridge
+# =============================================================================
+import torch.nn as nn
+from LION.models.LIONmodel import LIONmodel, ModelInputType, ModelParams
+import LION.CTtools.ct_geometry as ct
+
+
+class ARNetworkParams(ModelParams):
+    def __init__(
+        self,
+        n_channels: int,
+    ):
+        super().__init__(model_input_type=ModelInputType.IMAGE)
+        self.n_channels = n_channels
+
+
+class ARNetwork(LIONmodel):
```

Comment:
@Zakobian What do you think about naming? @cshoebridge and I were discussing how technically the model is not "AR", the framework for training is an AR, so we called this an ARNetwork. But I feel that maybe that is misleading and people call the netowkr also AR? Any opinions on naming?

Reply:
I would say that AR is indeed a training framework, but I would say that ambiguity in calling both the training framework and the method is ok here, since the improvements on the model so far have primarily been network architecture based, thus distinguishing themselves from AR? Otherwise they are all technically AR. Does this make sense?

Reply:
So then maybe lets call it AR @cshoebridge